src/getCoinCombination.test.js

```diff
@@ -7,4 +7,40 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return coorect combination for 2 cents`, () => {
+    const result = getCoinCombination(2);
+
+    expect(result).toEqual([2, 0, 0, 0]);
+  });
+
+  it(`should return coorect combination for 25 cents`, () => {
+    const result = getCoinCombination(25);
+
+    expect(result).toEqual([0, 0, 0, 1]);
+  });
+
+  it(`should return coorect combination for 99 cents`, () => {
+    const result = getCoinCombination(99);
+
+    expect(result).toEqual([4, 0, 2, 3]);
+  });
+
+  it(`should return coorect combination for 99.8 cents`, () => {
```

Comment:
There is a typo in the test description. It should be 'correct' instead of 'coorect'.